exporters/stats/stackdriver/README.md

```diff
@@ -94,5 +95,14 @@ StackdriverStatsExporter.createAndRegisterWithProjectId("MyStackdriverProjectId"
 Java 7 or above is required for using this exporter.
 
 ## FAQ
+### Why did I get a PERMISSION_DENIED error from Stackdriver when using this exporter?
+To use our Stackdriver Stats exporter, your Stackdriver account needs to have permission to [create
+custom metrics](https://cloud.google.com/monitoring/custom-metrics/creating-metrics), and that 
+requires a [Premium tier Stackdriver account](https://cloud.google.com/monitoring/accounts/tiers#this_request_is_only_available_in_the_premium_tier). 
+Please note that by default all new Stackdrvier accounts are Basic tier. To upgrade to a Premium 
+tier Stackdriver account, follow the instructions [here](https://cloud.google.com/monitoring/accounts/tiers#start-premium).
+
+In addition, if you've enabled Premium tier but still don't have access to custom metrics, please 
```

Comment:
This is covered in the Authentication section.

Reply:
I see, removed.